config/samples/external_secret_instana_agent_key.yaml

```
@@ -1,6 +1,7 @@
 apiVersion: v1
 stringData:
   key: xxx
+  key-2: yyy
```

The index needs to be adjusted, otherwise we end up with a broken deployment
```suggestion
  key-1: yyy
```
```
NAME                                       READY   STATUS                       RESTARTS   AGE
instana-agent-49fqw                        1/1     Running                      0          5m41s
instana-agent-cndqp                        1/1     Running                      0          5m37s
instana-agent-gqlg9                        1/1     Running                      0          5m39s
instana-agent-k8sensor-1-685bbc557-8jrjw   0/1     CreateContainerConfigError   0          27s
instana-agent-k8sensor-1-685bbc557-9httd   0/1     CreateContainerConfigError   0          27s
instana-agent-k8sensor-1-685bbc557-fc2dr   0/1     CreateContainerConfigError   0          27s
instana-agent-k8sensor-77b5659bbf-48nvs    1/1     Running                      0          28s
instana-agent-k8sensor-77b5659bbf-4fcsv    1/1     Running                      0          28s
instana-agent-k8sensor-77b5659bbf-bqjcn    1/1     Running                      0          28s
```

Once changed, everything works again.